src/nvim/os/shell.c

```diff
@@ -162,6 +163,8 @@ int os_call_shell(char_u *cmd, ShellOpts opts, char_u *extra_args)
 /// @param input The input to the shell (NULL for no input), passed to the
```

Comment:
`argv`: Yes. The example is also out of date with respect to that and the new `quote_cmd` argument.

`input`/`output`: Looks correct to me. What might be out of date?